test/src/main/java/org/apache/accumulo/test/ScanServerIT.java

```diff
@@ -359,7 +359,7 @@ protected static int ingest(AccumuloClient client, String tableName, int rowCoun
     return ingestedEntriesCount;
   }
 
-  protected static int getNumHostedTablets(AccumuloClient client, String tableId) throws Exception {
+  public static int getNumHostedTablets(AccumuloClient client, String tableId) throws Exception {
```

Comment:
Do you need this change?

Reply:
Good catch. I don't think so. I think this is left over from something I was trying out while developing the new test.

Reply:
Addressed in 6d821c4